molgenis-navigator/src/main/frontend/test/unit/utils/action.utils.js

```diff
@@ -0,0 +1,48 @@
+import { expect } from 'chai'
```

Comment:
The project contains multiple copies of the file, i suggest to put them in a single place